packages/suite/src/components/suite/StakingFeature.tsx

```diff
@@ -0,0 +1,57 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import { H3, P, variables } from '@trezor/components';
+
+const Circle = styled.div`
```

Comment:
100% agree, but wasn’t sure about it. There are some duplicated styled components across the app, so I thought it would be ok. Also, I haven’t found an appropriate place where to put reusable styled components, that don’t have any specific scope of use, like this `Circle`. So, I’ve created a styled file inside `src/components/suite`. Let me know if it’s OK with you.

https://github.com/everstake/trezor-suite/pull/1/commits/0ab37330a86b5d543f927db9918649711973f60b